APIv1.yaml

```diff
@@ -0,0 +1,927 @@
+                nodeType:
+                    type: string
+                permissionRead:
+                    type: boolean
+                permissionCreate:
+                    type: boolean
+                permissionUpdate:
+                    type: boolean
+                permissionDelete:
+                    type: boolean
+                permissionManage:
+                    type: boolean
+                sender:
+                    type: string
+                receiver:
+                    type: string
+                receiverDisplayName:
+                    type: string
+                receiverType:
+                    type: string
+                    items:
+                        - user
+                        - group
+                createdAt:
+                    type: string
+                lastEditAt:
+                    type: string
+        Columns:
+            type: array
+            maxItems: 1000
+            items:
+                $ref: "#/components/schemas/Column"
+        Column:
+            type: object
+            required:
+                - id
+                - title
+                - tableId
+                - mandatory
+                - type
+            properties:
+                id:
+                    type: integer
+                tableId:
+                    type: integer
+                createdBy:
+                    type: string
+                createdAt:
+                    type: string
+                lastEditBy:
+                    type: string
+                lastEditAt:
+                    type: string
+                type:
+                    type: string
+                subtype:
+                    type: string
+                mandatory:
+                    type: boolean
+                description:
+                    type: string
+                orderWeight:
+                    type: integer
+                numberDefault:
+                    type: number
+                numberMin:
+                    type: number
+                numberMax:
+                    type: number
+                numberDecimals:
+                    type: integer
+                numberPrefix:
+                    type: string
+                numberSuffix:
+                    type: string
+                textDefault:
+                    type: string
+                textAllowedPattern:
+                    type: string
+                textMaxLength:
+                    type: integer
+                selectionOptions:
+                    type: string
+                selectionDefault:
+                    type: integer
+                datetimeDefault:
+                    type: string
+        Rows:
+            type: array
+            maxItems: 1000
+            items:
+                $ref: "#/components/schemas/Row"
+        Row:
+            type: array
+            description: Values depending on the table definition.
+            items: {}
```

Comment:
I assume we could be more specific here - it's not just any object, but it has `columnId` and `value` properties like in the web app, no? Otherwise mapping the values to the actual rows would be quite hard :wink: 

![image](https://user-images.githubusercontent.com/4741199/228588944-2e75481a-b1af-42d5-af4b-43141ccdd17b.png)